apps/native-component-list/src/screens/MailComposerScreen.tsx

```diff
@@ -61,7 +61,11 @@ export default function MailComposerScreen() {
           <Text>{label}</Text>
           <Button
             onPress={Platform.select({
-              android: () => packageName && openApplication(packageName),
+              android: () => {
```

Comment:
Is this change necessary? Not sure if there's any different in terms of functionality between using the `if` statement here and the previous `&&` usage, since the `ios` below is still using `&&`.

Reply:
Yes, this change is necessary because I changed the interface from implicit `any` to `void` in `packages/expo-intent-launcher/src/IntentLauncher.ts` so it shows correctly in the docs. This caused TypeScript to throw an error here which I resolved with this change.